packages/dx-react-grid-bootstrap3/src/templates/paging-panel/pagination.test.jsx

```diff
@@ -0,0 +1,93 @@
+import React from 'react';
+import { mount, shallow } from 'enzyme';
+import { Pagination } from './pagination';
+
+const defaultProps = {
+  totalPages: 10,
+  currentPage: 1,
+  totalCount: 96,
+  pageSize: 10,
+  getMessage: key => key,
+  onCurrentPageChange: () => {},
+};
+
+describe('Pagination', () => {
+  it('can show info about rendered pages', () => {
+    const getMessage = jest.fn(key => key);
+    const tree = shallow((
+      <Pagination
+        {...defaultProps}
+        getMessage={getMessage}
+      />
+    ));
+
+    expect(getMessage)
+      .toBeCalledWith('info', { from: 11, to: 20, count: 96 });
+    expect(tree.find('span > span').text())
+      .toBe('info');
+  });
+
+  it('can render pagination arrows', () => {
+    const onCurrentPageChange = jest.fn();
+    const arrows = mount((
+      <Pagination
+        {...defaultProps}
+        onCurrentPageChange={onCurrentPageChange}
+      />
+    )).find('.pager li');
+
+    const prew = arrows.at(0);
+    const next = arrows.at(1);
+
+    prew.find('a').simulate('click');
+    next.find('a').simulate('click');
+
+    expect(arrows).toHaveLength(2);
+    expect(prew.hasClass('disabled')).toBeFalsy();
+    expect(next.hasClass('disabled')).toBeFalsy();
+    expect(onCurrentPageChange.mock.calls).toHaveLength(2);
+  });
+
+  it('disables the prev arrow if current page is 1', () => {
```

Comment:
`disables the prev arrow if the current page is first` because you write - 1 but the currentPage prop = 0